bsip-0071.md

```diff
@@ -122,6 +126,10 @@ FP<sub>M</sub> is still used for determining if a debt position is margin called
 
 **Note:** when comparing prices, `max` is to be understood in terms of debt divided by collateral.
 
+#### Adjustment to voting power
+
+The voting power for BTS backed SmartCoins added to the owner of a margin position is reduced. Only the exccess collateral counts as voting power. Example: If a position has CR 1.8, the voting power from this position only counts the BTS that represent the excess of 0.8. Reasoning for this is given in the section Discussion / Voting power.
+
```

Comment:
Changes about voting power should not be in the scope of this BSIP.

Reply:
This BSIP cements the ability for risk-free margin to the max, multiplying voting power, and setting a status quo and "confirmed by core". In the worst case scenario, this is an irreversible action.

Planning for the worst-case scenario, I see it very much in relevance to this BSIP, and would rather not want to remove it. This is certainly an effect that we already see now, still having it inside the core is of different magnitude IMO.

Reply:
I totally disagree with the adjustment to voting power, voting power is inside BTS, one should own the voting power of the BTS in his account, either his account is debt free or not, debt just means he would like to bear the relevant risk, no others.

Reply:
What risk?

Reply:
@bitcrab If you don't own the associated bitasset for your debt, then you effectively sold that 1x CR BTS which the bitasset holder can claim at any moment. Anything above 1x CR would still be yours to vote with, so just maintain a good CR 👍